ffi/src/lib.rs

```diff
@@ -13,6 +16,9 @@ pub mod owned;
 mod primitives;
 pub mod slice;
 
+/// A specialized `Result` type for FFI operations
+pub type Result<T> = core::result::Result<T, FfiReturn>;
```

Comment:
I'm guessing that this is why you renamed `FfiResult` to `FfiReturn`.

Reply:
yeah